### PR TITLE
Add grid, toast and rating

### PR DESCRIPTION
**Pull request by yash9263**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1777,11 +1783,138 @@ <h1 class="header-1">
...
+          <div class="docs-code-example">
+          <pre><code class="html">
+<span class="ratings">
+  <i class="bx bxs-star"></i>
+  <i class="bx bxs-star"></i>
+  <i class="bx bxs-star"></i>
+  <i class="bx bx-star"></i>
+  <i class="bx bx-star"></i>
+</span>
+          </code></pre>
```
Comment (gauravsinhaweb) [inline index.html:1813]: improve indentation!

File: index.html [hunk 2]
```diff
@@ -1777,11 +1783,138 @@ <h1 class="header-1">
...
+            <div class="docs-code-example">
+            <pre><code class="html">
+<div class="alert alert-primary flex-center-y alert-dismissible" role="alert">
+  <i class="bx bxs-info-circle alert-icon primary-icon"></i>
+  Hello There! I am a toast box.
+  <button class="dismiss-btn">
+    <i class="bx bx-x dismiss-icon"></i>
+  </button>
+</div>
+            </code></pre>
```
Comment (gauravsinhaweb) [inline index.html:1847]: improve indentation.

File: styles/components/grid.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.si-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 8px;
+}
+.si-col {
+}
```
Comment (gauravsinhaweb) [inline styles/components/grid.css:7]: remove unnecessary class.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes!

File: index.html (hunk 1 above)
Comment (yash9263) [reply index.html:1813]: the issue with highlight.js it keeps the indentation inside the code and it overflows will update it soon

File: index.html [hunk 4]
```diff
@@ -1777,11 +1783,138 @@ <h1 class="header-1">
...
+            <div class="alert alert-primary flex-center-y alert-dismissible" role="alert">
+              <i class="bx bxs-info-circle alert-icon primary-icon"></i>
+              Hello There! I am a toast box.
+              <button class="dismiss-btn">
+                <i class="bx bx-x dismiss-icon"></i>
+              </button>
+            </div>
+            </div>
+            <div class="docs-code-example">
+            <pre><code class="html">
```
Comment (MayankKumar10) [inline index.html:1839]: use different class name

File: index.html [hunk 5]
```diff
@@ -1777,11 +1783,138 @@ <h1 class="header-1">
...
+        </section>
+        <section id="grid">
+          <article class="docs-article">
+            <h1 class="header-1">
+              <!-- Component Name -->
+              Grid
+            </h1>
+            <p class="docs-text">
+              <!-- Component explaination text -->
+              Grids are an essential tool for building any modern ui. You can use our grid classes to distribute your elements into grids to create layouts as required.
```
Comment (MayankKumar10) [inline index.html:1859]: use prettier for indentation

File: styles/helpers/utilties.css [hunk 6]
```diff
@@ -2,6 +2,18 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.flex-center-x {
+  display: flex;
+  justify-content: center;
+}
+
```
Comment (pankaj-prog) [inline styles/helpers/utilties.css:16]: Nice to see that variables and classes are descriptive.

File: styles/components/grid.css [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+.filler {
+  height: 48px;
```
Comment (MayankKumar10) [inline styles/components/grid.css:10]: If not necessary try to avoid using px instead use rem.

File: styles/components/navbar.css [hunk 8]
```diff
@@ -10,7 +10,8 @@
 }
 .ex-nav-items {
   display: flex;
```
Comment (MayankKumar10) [inline styles/components/navbar.css:12]: If you used these flex properties elsewhere you can make it a helper class.

**Review by MayankKumar10 — APPROVED** [review]
Look into these everything looks good you can merge it,

**Review by pankaj-prog — APPROVED** [review]
I couldn't find anything for suggestion. Instead I learned from reviewing your code about using comments, descriptive class names and using variables where possible.

**Pull request merged**